src/js/tracks/text-track.js

```diff
@@ -227,7 +226,7 @@ TextTrack.prototype.removeCue = function(removeCue) {
 /*
  * Downloading stuff happens below this point
  */
-let parseCues = function(srcContent, track) {
+var parseCues = function(srcContent, track) {
   if (typeof window['WebVTT'] !== 'function') {
```

Comment:
Yeah, can't use a let var before it's defined in the file. How about I make it `function parseCues()`? I don't like burying the constructor in its own file.